packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1084,7 +1084,7 @@ async function handler(
             await sendRescheduledEmails({
               ...copyEvent,
               additionalNotes, // Resets back to the additionalNote input and not the override value
-              cancellationReason: "$RCH$" + rescheduleReason ? rescheduleReason : "", // Removable code prefix to differentiate cancellation from rescheduling for email
+              cancellationReason: "$RCH$" + (rescheduleReason ? rescheduleReason : ""), // Removable code prefix to differentiate cancellation from rescheduling for email
```

Comment:
+ has higher precedence then "?". So, rescheduleReason was being sent and not $RCH + rescheduleReason.